packages/nextjs/src/server-helpers.server.ts

```diff
@@ -0,0 +1,17 @@
+import { auth } from './app-router/server/auth';
+import { currentUser } from './app-router/server/currentUser';
+import { authMiddleware } from './server/authMiddleware';
+import { clerkClient } from './server/clerkClient';
+import { getAuth } from './server/getAuth';
+import { withClerkMiddleware } from './server/withClerkMiddleware';
```

Comment:
❓ Should we mark withClerkMiddleware as deprecated? Do we want users to keep using it?

Reply:
Added JS343, we will deprecated once testing is over. Thanks!